internal/command/client.go

```diff
@@ -148,13 +172,21 @@ func (g *gcp) GetAccessToken(ctx context.Context) (string, error) {
 		return "", fmt.Errorf("%w: failed to fetch token from GCP ADC token source: %w", errTokenFetchFailure, err)
 	}
 
-	log.FromContext(ctx).Info("fetched token using GCP ApplicationDefaultCredential")
+	log.Info("fetched token using GCP ApplicationDefaultCredential")
+
+	payload, _ := idtoken.ParsePayload(token.AccessToken)
+
+	prettyPayload, _ := json.MarshalIndent(payload.Claims, "", "  ")
+
+	log.Info(fmt.Sprintf("Google OIDC ID token payload: %s", prettyPayload))
```

Comment:
Do want to point out this line will log the ID token claims to the app. This is to help customers identify the claims they will want to map in Command. I don't think any of these claims are confidential. Let me know if there's any disagreements with this.